.github/workflows/lint-format.yml

```diff
@@ -25,6 +25,9 @@ jobs:
       - name: Install nightly toolchain
         run: cargo make install-nightly
 
+      - name: Install stable toolchain
+        run: cargo make install-stable
+
```

Comment:
if we add this we should remove nightly?

Reply:
idk do we use both or nah? check the makefile

Reply:
we need both. some features we use for format are not in stable 1.79